tests/Backend/Workspaces/ParallelWorkspacesTestCase.php

```diff
@@ -0,0 +1,131 @@
+<?php
+namespace Keboola\Test\Backend\Workspaces;
+
+use Keboola\Db\Import\Snowflake\Connection;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Options\TokenCreateOptions;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\StorageApiTestCase;
+
+abstract class ParallelWorkspacesTestCase extends StorageApiTestCase
+{
+    /** @var string */
+    protected $runId;
+
+    /** @var Client */
+    protected $workspaceSapiClient;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->initEmptyTestBucketsForParallelTests();
+
+        $description = get_class($this) . '\\' . $this->getName();
+
+        $this->deleteOldTestWorkspaces($description);
+
+        $this->workspaceSapiClient = new Client([
```

Comment:
Takhle vytvořený klient nemá useragent nastavený jako ten normální (viz \Keboola\Test\ClientTestCase::getClient)

Reply:
Jo to upravím. Vybral jsem ten kód z letního PR kdy jsme ještě UserAgenta per test neměli a tak nějak jsem zapomněl že to tam bylo dodělaný.

Reply:
Mělo by být fixed